x-pack/qa/oidc-op-tests/src/javaRestTest/java/org/elasticsearch/xpack/security/authc/oidc/C2IdOpTestCase.java

```diff
@@ -103,7 +220,9 @@ protected static void registerClients(String... jsonBody) throws IOException {
             SocketAccess.doPrivileged(() -> {
                 for (HttpPost httpPost : requests) {
                     try (CloseableHttpResponse response = httpClient.execute(httpPost, context)) {
-                        assertThat(response.getStatusLine().getStatusCode(), equalTo(201));
+                        assertBusy(() -> assertThat(response.getStatusLine().getStatusCode(), equalTo(201)));
```

Comment:
We need an `assertBusy` here because the actual OIDC service may not yet be available. See my note in the [PR description](https://github.com/elastic/elasticsearch/pull/96756#issue-1752182307)